packages/usa-character-count/src/usa-character-count.stories.js

```diff
@@ -1,4 +1,5 @@
 import Component from "./usa-character-count.twig";
+import disabledOption from "./index.json";
```

Comment:
What do you think about changing the two booleans into a single radio control like we do in the Date range picker stories? It makes it easier to isolate the testing so that you are only checking one attribute at a time. 

I also have a slight lean towards making separate stories for Disabled and Aria disabled for all our components like we have for Date Range Picker. This might just be a personal preference though.

Reply:
The reason I originally decided to have them be separate booleans instead of the radio buttons, is because `aria-disabled` doesn't apply the disabled _state_ to the components. You are still able to set text in inputs/text fields, dropdowns can be opened, checkboxes can be checked/unchecked.

My thought was this would allow users to test them separately _and together_ and that having them in conjunction would allow them to be selected by the screen reader while being actively disabled and not interactive.

>  ...the aria-disabled="true" only semantically exposes these elements as being disabled. Web developers must manually ensure such elements have their functionality suppressed when they are exposed in the disabled state.

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-disabled

It does appear that [some users think that you do not need both at this time](https://stackoverflow.com/questions/38059140/when-to-use-the-disabled-attribute-vs-the-aria-disabled-attribute-for-html-eleme#:~:text=You%20will%20be%20happy%20to%20know%20that%20both%20NVDA%20and%20JAWS%20skip%20the%20field%20(or%20if%20explicitly%20focused%2C%20announce%20that%20is%20disabled).), as `disabled` seems to accept keyboard focus for screen readers (at least in modern browsers / our storybook example).

I'm definitely open to suggestions on what we think is the best path moving forward. You might be much more familiar with aria attributes than I am so I'm happy to change this back to the radio button slike date range picker if you think that makes more sense!